things/service/src/test/java/org/eclipse/ditto/things/service/enforcement/MergeThingCommandEnforcementTest.java

```diff
@@ -119,6 +121,44 @@ void rejectByPolicy(final TestArgument arg) {
                 () -> ThingCommandEnforcement.authorizeByPolicyOrThrow(policyEnforcer, arg.getMergeThing()));
     }
 
+    @Test
+    void acceptUsingRegexInPolicy() {
+        final TestArgument testArgument = TestArgument.of("/",
+                Set.of("/", "/attributes/complex"),
+                Set.of("/attributes/simple")
+        );
+        final TrieBasedPolicyEnforcer policyEnforcer = TrieBasedPolicyEnforcer.newInstance(testArgument.getPolicy());
+        final JsonObject patch = JsonObject.newBuilder()
+                .set("attributes", JsonObject.newBuilder()
+                        .set("complex", JsonObject.newBuilder()
+                                .set("{{ /.*/ }}", JsonValue.nullLiteral())
+                                .build()
+                        )
+                        .set("another", 42)
+                        .build())
+                .build();
+        final MergeThing mergeThing = MergeThing.of(TestSetup.THING_ID, JsonPointer.empty(), patch, TestArgument.headers());
+        final MergeThing authorizedMergeThing =
+                ThingCommandEnforcement.authorizeByPolicyOrThrow(policyEnforcer, mergeThing);
+        assertThat(authorizedMergeThing.getDittoHeaders().getAuthorizationContext()).isNotNull();
+    }
+
+    @Test
+    void rejectUsingRegexWithRevokesInPolicy() {
+        final TestArgument testArgument = TestArgument.of("/", Set.of("/"), Set.of("/attributes/complex/nested/secret"));
+        final TrieBasedPolicyEnforcer policyEnforcer = TrieBasedPolicyEnforcer.newInstance(testArgument.getPolicy());
+        final JsonObject patch = JsonObject.newBuilder()
+                .set("attributes", JsonObject.newBuilder()
+                        .set("{{ /.*/ }}", JsonValue.nullLiteral())
```

Comment:
Yes, it is .. in order to avoid strange "sideeffects" where using the "regex" syntax to e.g. set the same value for a lot of matched JSON fields.
I explicitly only wanted to add support for deleting existing JSON fields - also documented that way.